modules/tests/src/test/scala/dev/profunktor/redis4cats/TestScenarios.scala

```diff
@@ -462,8 +462,18 @@ trait TestScenarios { self: FunSuite =>
     } yield ()
   }
 
-  def connectionScenario(redis: RedisCommands[IO, String, String]): IO[Unit] =
-    redis.ping.flatMap(pong => IO(assertEquals(pong, "PONG"))).void
+  def connectionScenario(redis: RedisCommands[IO, String, String]): IO[Unit] = {
+    val clientName = "hello_world"
+    for {
+      _ <- redis.ping.flatMap(pong => IO(assertEquals(pong, "PONG"))).void
+      _ <- redis.getClientName().flatMap(name => IO(assertEquals(name, None)))
+      _ <- redis
```

Comment:
Can you split the `flatMap` call using the for-comprehension? You can do it for all the calls if you want. It was alright when it was in one line, but now it isn't 😄 

```scala
n <- redis.setClientName(clientName)
_ <- IO(assert(result, s"Failed to set client name: '$clientName'"))
```

Reply:
Sure can :smile: